### ISSUE TITLE
[Question] Can We Marge Two Label Studio together?

### ISSUE THREAD

**Issue by Msameim181** [heartexlabs/label-studio#3201]
[Question] Can We Marge Two Label Studio together?
Let's make clear things for you. 
1. Imagine we have a "local" LabelStudio on our system (let's say Windows 10). 
2. And then we need to collaborate with someone else, why? because we start a big project and did a lot of work on it but now we ran out of time or etc. and need help. So What Should we do? (Here is A BIG and FIRST Question for you, What should we do?)
3. An idea is, We lunch a VPS and LabelStudio on it. But there are two ways to do this.
3.1. If We Just import the Data in normal ways we can just replace our LabelStudio Folder (in our VPS) with LabelStudio Folder (in our Win System). So then we have everything on VPS and We can resume everything.
3.2. But What if we import the data using "Local Storage" and gave a full path of our data in the windows system? In this case, how should we act? Maybe you would say start There (VPS) Again and then Marge it. BUT HOW?

So in summary:

- How can we Move a Local LabelStudio to a VPS and Make it Public?
- How can we Export the whole Project and Then Import it to another LabelStudio?
- How can we export annotations and (Fix the name of data) Import it to the new project or Other LabelStudio?
- How can we marge Two LabelStudio?

P.S: I'd be pleased to help you improve your system for better use, experience, and features. Especially about these questions.

Be Safe.

**Comment by makseq** [comment]
1. https://github.com/heartexlabs/label-studio#deploy-in-a-cloud-instance - you can use this one deploy button to run LS in Google Cloud. It will be public automatically. 
2. I would recommend that you use postgres DB, otherwise your data in the cloud won't be persistent (from time to time it will be reset). 
3. There is no way to export the whole project, but you can export tasks + annotations in the Data manager and then re-import them in another LS instance:
  3.1. Go to Project => Data manager page, click Export button, get a JSON file 
  3.2. Go to Project => Settings => Labeling Interface, copy the labeling config
  3.3. Create a new project in the cloud, insert the old labeling config and import the previously exported JSON
